gRPC-ProtoRPC.podspec

```diff
@@ -61,7 +61,7 @@ Pod::Spec.new do |s|
 
   s.dependency 'gRPC', version
   s.dependency 'gRPC-RxLibrary', version
-  s.dependency 'Protobuf', '~> 3.0.0-beta-3.1'
+  s.dependency 'Protobuf', '~> 3.0'
```

Comment:
The template for this file seems no longer exists. Update the comments at the beginning of this file?